activemodel/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+*   Validate multiple contexts on `valid?` and `invalid?` at once.
+
+    Example:
+    
+        class Person
+          include ActiveModel::Validations
+    
+          attr_reader :name, :title
+          validates_presence_of :name, on: :create
+          validates_presence_of :title, on: :update
+        end
+    
+        person = Person.new
+        person.valid?([:create, :update])    # => true
```

Comment:
Not many changes should be made, but the interface of the `valid?` should be changed to `valid?(*)` in activemodel/activerecord implementations (and everywhere where is `valid?` is used should be changed as well, like `save`, `invalid?` and so on). It can brake some gems. 

In the beginning I went by this path, but then after changing a lot of places in the code, I found that using an array actually can fit better to whole conception, because it makes sense to pass an array instead of passing an array through a splat; passing a simple array without splat will look more clean in many cases.